portal-web/test/functional/com/liferay/portalweb/paths/portlet/claysample/ClaySamplePortlet.path

```diff
@@ -36,6 +36,26 @@
 	<td>//input[@type='checkbox' and @id='customId' and contains(@class,'custom-css-class')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CHECKBOX_SELECTABLE_IMAGE_CARD</td>
+	<td>//div[contains(@id,'image-card-icon-block')]//input[contains(@class,'custom-control-input')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CARD_ACTIVE</td>
+	<td>//div[contains(@class,'active file-card form-check-card form-check form-check-top-left card-type-asset')]</td>
```

Comment:
I don't think we need all the classes. Maybe shorten to: `//div[contains(@class,'active file-card')]`